routes/index.js

```diff
@@ -153,7 +158,9 @@ router.post(`/${APP_VERSION}/download`, async function (req, res, next) {
 
       //iterate over the result sets and create a work sheet to append to the book
       Object.keys(gqlRes).map((k) => {
-        const ws = XLSX.utils.json_to_sheet(gqlRes[k])
+        const ws = xlsxHeader
+          ? XLSX.utils.json_to_sheet(gqlRes[k], { header: xlsxHeader })
+          : XLSX.utils.json_to_sheet(gqlRes[k])
         XLSX.utils.book_append_sheet(wb, ws, k)
```

Comment:
same here

Reply:
json_to_sheet is the external library, we can't change it